flink-java/src/test/java/org/apache/flink/api/java/utils/OptionsTest.java

```diff
@@ -54,16 +54,6 @@ public void testChoicesWithValidDefaultValue() {
         Assert.assertEquals(option.getDefaultValue(), "a");
     }
 
-    @Test
-    public void testChoicesWithInvalidDefautlValue() throws RequiredParametersException {
```

Comment:
It's redundant

Reply:
But the test code is different from `testChoicesWithInvalidDefaultValue`.

Reply:
Then change the name?  testChoicesWithInvalidDefaultValue**Second**().
Or, merge them into one test?

Reply:
I would suggest to change the method name to `testChoicesWithInvalidDefaultValue2` and put it after `testChoicesWithInvalidDefaultValue`. 

`expectedException` can't be used in a test method twice.

Reply:
fixed